_targets.R

```diff
@@ -81,202 +81,208 @@ gagesii$ID <- substr(gagesii$ID, start=2, stop=nchar(gagesii$ID))
-  tar_target(p1_FDC_metrics_season_high,
-             calc_FDCmetrics(site_num = p1_screened_site_list_season_high, 
-                             clean_daily_flow = p1_clean_daily_flow_season_high, 
-                             yearType = yearType,
-                             drainArea_tab = NULL,
-                             NE_probs = NE_quants,
-                             seasonal = TRUE,
-                             season_months = season_months_high,
-                             stat_type = 'POR',
-                             year_start = season_year_start_high,
-                             out_format = 'pivot'),
-             map(p1_screened_site_list_season_high)),
-
-  ########moving window nonstationarity stuff
-   ##table with all the FDC metrics computed on a moving window. The parameter min_yrs_in_window
-  ##screens out any moving windows for which there are too few years to be reliable. Can be an issue 
-  ##when there are large gaps in the data record because a 20 year window might only have a few years of 
-  ##actual data.The yr_ct column indicates how many complete years were in the window just to keep track of it.
-
-   tar_target(p1_moving_window_metrics,
-              calc_moving_window_metrics(site_num = p1_screened_site_list,
-                                         window_length = window_length,
-                                         increment = increment,
-                                         min_yrs_in_window = min_yrs_in_window,  
-                                         clean_daily_flow = p1_clean_daily_flow,
-                                         yearType = yearType,
-                                         drainArea_tab = p1_drainage_area,
-                                         NE_probs = NE_quants,
-                                         digits = 3, seasonal = FALSE,
-                                         year_start = year_start),
-             map(p1_screened_site_list)),
-  
-  ##screen out any sites that don't have enough moving windows to plot (min_windows)
-  ##using 10 for a default
-  tar_target(p1_screened_plot_sites,
-             screen_plot_sites(moving_window_metrics = p1_moving_window_metrics,
-                               min_windows = min_windows)),
-  
-  tar_target(p1_moving_window_plots,
-             make_plots_by_site(site = p1_screened_plot_sites,
-                                moving_window_metrics = p1_moving_window_metrics,
-                                window_length = window_length,
-                                outdir = "1_fetch/out/stationarity_plots"),
-             map(p1_screened_plot_sites),
-             format = "file"),
-  
-  tar_target(p1_moving_window_summary_plots,
-             plot_trend_summary(moving_window_metrics = p1_moving_window_metrics,
-                                screened_plot_sites = p1_screened_plot_sites,
-                                outdir = "1_fetch/out/stationarity_plots"),
+  tar_target(p1_has_data_check,
```

Comment:
Instead of needing to comment out targets, you can run `tar_make(<target name>)` to run only the target you want and its dependencies